src/test/java/api/support/fakes/FakeCQLToJSONInterpreter.java

```diff
@@ -40,7 +41,16 @@ public FakeCQLToJSONInterpreter(boolean diagnosticsEnabled) {
     }
   }
 
-  private Comparator<? super JsonObject> sortForQuery(String sort) {
+  private Comparator<JsonObject> sortForQuery(String sort) {
```

Comment:
Please can a test to demonstrate that the fake can sort by multiple clauses

Reply:
Done